### ISSUE TITLE
Discord invite link is not valid

### ISSUE THREAD

**Issue by sebastianneubert** [flarum/framework#3762]
### Current Behavior

in the README.md is a link to a [Discord server](https://flarum.org/discord/). This link redirects to an invalid invitation. So nobody can join the Discord.

### Steps to Reproduce

1. Click on the link in the README.md
2. See a "Invalid invitation link" message from Discord

### Expected Behavior

1. Click on the link and see a welcome message

### Environment

- README.md in root dir

### Output of `php flarum info`

```
Output of "php flarum info", run this in terminal in your Flarum directory.
```

### Possible Solution

update the redirect to a valid invitation link

**Comment by SychO9** [comment]
It's not invalid though 🤔 ?

![image](https://user-images.githubusercontent.com/20267363/225259352-64398f6c-dc78-4dcd-8de7-d68e2db2471b.png)

**Comment by davwheat** [comment]
Is it possible that the user may have been banned for some reason?

I would suggest trying incognito @sebastianneubert, and if that works, let us know your Discord username and we can see if you're banned from the server for any reason.

**Comment by sebastianneubert** [comment]
Hmm, strange. So what I did to fix this:

- use the ingognito mode of the browser. that works, but i need to register a new user and i cant use the given one
- get the link from the redirect directly and add the server manually in the discord app -> that works

maybe there is an issue with that redirect. the link I used was valid and the invite was accepted after serveral tries (see above).

Thanks in advance. I'll catch you guys in the discord server
